Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN chmod +x run.sh
 RUN npm install
 RUN npm run build-ts
 
+RUN apk update && apk add bash libbz2 xz-dev libffi-dev openssl-dev python3 build-base python3-dev py3-pip
```

Comment:
What are these for?

Reply:
Required for fastqutils bam conversion.

Reply:
Didn't this already work though?

Reply:
Some python dependencies are needed but we didn't get enough time to test which can be removed. I'll look into removing which is not needed.